internal/ecs/metadata/metadata_test.go

```diff
@@ -63,3 +63,82 @@ func TestClusterToClusterName(t *testing.T) {
 		assert.Equal(t, testCase.expectedClusterName, clusterName)
 	}
 }
+
+func TestTaskMetadataEndpoint(t *testing.T) {
+	v4 := "http://localhost/v4"
```

Comment:
t.parallel?

Reply:
also applies to other tests I believe

Reply:
I tried but `t.Setenv` cannot be called in parallel tests
```
--- FAIL: TestTaskMetadataEndpoint (0.00s)
    --- FAIL: TestTaskMetadataEndpoint/returns_v4_endpoint_if_v3_and_v4_exists (0.00s)
panic: testing: t.Setenv called after t.Parallel; cannot set environment variables in parallel tests [recovered]
	panic: testing: t.Setenv called after t.Parallel; cannot set environment variables in parallel tests
```